src/python/model.hpp

```diff
@@ -132,6 +132,10 @@ namespace se3
       .add_property("fix_hasVisual", bp::make_function(&ModelPythonVisitor::fix_hasVisual, bp::return_internal_reference<>())  )
       .add_property("fix_bodyNames", bp::make_function(&ModelPythonVisitor::fix_bodyNames, bp::return_internal_reference<>())  )
 
+      .def("getFrameParentJoint", &ModelPythonVisitor::getFrameParent)
+      .def("getFramePositionInParent", &ModelPythonVisitor::getJointToFrameTransform)
```

Comment:
a good name here may just be getFramePlacement (it is implicit that it is regarding to the parent joint)